### ISSUE TITLE
Tatin Version 0.106.1 doesn't expose Tatin user commands anymore

### ISSUE THREAD

**Issue by Alexander-Block** [aplteam/Tatin#189]
After upgrading Tatin to the current version 0.106.1 I cannot use any Tatin user commands anymore. This issue can be reproduced as follows. First I reset my Tatin installation using `]Tools.Activate tatin -reset`. 

Then I start a new Dyalog APL 19.0 session and run the following:

```APL
      ]Tatin.Version
 Tatin  0.104.1+1761  2024-01-28 

      ]Tatin.UpdateTatin

Your version of Tatin is 0.104.1
Latest published version is 0.106.1
Would you like to download and install 0.106.1 ? (Y/n) Y
Tatin successfully updated to version 0.106.1 in C:\Users\<⎕AN>\Documents\MyUCMDs\Tatin
```

As an aside `UpdateTatin` prints the wrong installation path. There is no folder "C:\Users\<⎕AN>\Documents\MyUCMDs\Tatin" on my system, Tatin is installed under "C:\Users\<⎕AN>\Documents\Dyalog APL-64 19.0 Unicode Files\SessionExtensions\CiderTatin\Tatin" and I can check that version 0.106.1 is correctly transferred there. 

When starting a new Dyalog APL 19.0 session afterwards I see the following: 

```APL
     ⎕SE.Tatin.Version
 Tatin  0.106.1+1802  2024-03-18 

      ]Tatin -?
]tatin could be one of:           
                                  
      ]CIDER.AddTatinDependencies 
      ]CIDER.ListTatinDependencies

      ]Tatin.Version
* Invalid user command; to see a list of all user commands type
      ] -?                                                     
```

Now all Tatin user commands seem to be missing, they are usable again when resetting the Tatin version to 0.104.1.

I think this might be a side effect of the recent changes to Tatin.dyalog.

**Comment by aplteam** [comment]
At the moment it's a mystery why that happens. There is nothing wrong with the changes made to `Tatin.dyalog`, which becomes evident when one executes
```
]ureset
```
because then all the Tatin user commands are available again, so this serves as a workaround.

**Comment by aplteam** [comment]
Fixed in 0.106.2
